response_operations_ui/contexts/case.py

```diff
@@ -0,0 +1,35 @@
+from flask import url_for
+
+
+def build_response_status_context(
+    ru_ref: str,
+    survey_short_name: str,
+    case_group_id: str,
+    ce_period: str,
+    allowed_transitions_for_case: dict,
+    survey_id: str,
+    has_reporting_unit_permission: bool,
+) -> dict:
+    if has_reporting_unit_permission:
```

Comment:
You have kind of doubled up here with the frontend and checked permissions twice ({% if isReportingUnitPermission %} in the template), so else is never reachable. However we shouldn't need to do isReportingUnitPermission in the template, as it's been done in the back end. Depends on what else uses it on how to pass it through, if it's just this then checking change_response_status is not null would work

Reply:
I've removed this permissions check from the frontend and made it just check for it existing